src/__tests__/operations/property-filter.test.ts

```diff
@@ -7,13 +7,13 @@ const propertyFiltering = {
   filteringProperties: [
     {
       key: 'id',
-      operators: [':', '!:'],
+      operators: [':', '!:', '^'],
       groupValuesLabel: 'Id values',
       propertyLabel: 'Id',
     },
     {
       key: 'field',
-      operators: [':', '!:'],
+      operators: [':', '!:', '^', '?'],
```

Comment:
What is this `?` operator?

Did you mean to test how unsupported operator would work? 

At least, there should be a clarifying comment, or ideally, the actual separate test case with a descriptive name, for example, "handles unsupported operators"